pkg/Microsoft.Windows.Compatibility/test/Microsoft.Windows.Compatibility.Validation.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit: I would call the directory tests like others.

Reply:
Fixed.